com.unity.render-pipelines.universal/Runtime/NativeRenderPass.cs

```diff
@@ -579,12 +579,12 @@ internal static Hash128 CreateRenderPassHash(RenderPassDescriptor desc, uint has
             return CreateRenderPassHash(desc.w, desc.h, desc.depthID, desc.samples, hashIndex);
         }
 
-        private static RenderPassDescriptor InitializeRenderPassDescriptor(CameraData cameraData, ScriptableRenderPass renderPass)
+        private RenderPassDescriptor InitializeRenderPassDescriptor(CameraData cameraData, ScriptableRenderPass renderPass)
         {
             var w = (renderPass.renderTargetWidth != -1) ? renderPass.renderTargetWidth : cameraData.cameraTargetDescriptor.width;
             var h = (renderPass.renderTargetHeight != -1) ? renderPass.renderTargetHeight : cameraData.cameraTargetDescriptor.height;
             var samples = (renderPass.renderTargetSampleCount != -1) ? renderPass.renderTargetSampleCount : cameraData.cameraTargetDescriptor.msaaSamples;
-            var depthID = renderPass.depthOnly ? renderPass.colorAttachment.GetHashCode() : renderPass.depthAttachment.GetHashCode();
+            var depthID = renderPass.depthOnly ? renderPass.colorAttachment.GetHashCode() : renderPass.overrideCameraTarget ? renderPass.depthAttachment.GetHashCode() : m_CameraDepthTarget.GetHashCode();
             return new RenderPassDescriptor(w, h, samples, depthID);
```

Comment:
nitpick: personally I would split this line as I am not a big fan of nested ternary operators for readability/debugging

Reply:
fixed it